### DISCUSSION TITLE
Possible to access the topocentric transformation ?

### DISCUSSION THREAD

**Discussion by stefangachter** [pyproj4/pyproj#1253] [Q&A] [answered]
Dear all,
I am converting cartesian geographic coordinates to topocentric ones as described here
https://proj.org/operations/conversions/topocentric.html
I have not only positions but also velocities to convert including the associated uncertainty covariances.
Converting the velocities could be done the same way as positions. But I am not aware if the covariances could be converted with pyproj. 
However, if I can access the translation vector and rotation matrix of the geographic to topocentric conversion, then I could do it "manually".
Is it possible to access the transformation, e.g. as 4x4 isomorph transformation matrix? Or, any suggestion how to convert "uncertainties" with pyproj?

**Comment by snowman2** [comment]
Related: https://github.com/pyproj4/pyproj/discussions/863

**Reply by snowman2** [reply]
I recommend asking this question in the PROJ mailing list: https://github.com/OSGeo/PROJ/

**Comment by stefangachter** [comment, marked as answer]
Raised the question here https://github.com/OSGeo/PROJ/issues/3666
